cpanfile

```diff
@@ -11,6 +11,7 @@ requires 'ExtUtils::MakeMaker', '6.72'; # [1]
 requires 'Guard', '1.023';
 requires 'PadWalker', '2.2';
 requires 'aliased', '0.34';
+requires 'Net::SSLeay', '1.81';
```

Comment:
Woo, this seems to install cleanly on macOS! Now seems a good opportunity to upgrade `Net::Twitter::Lite::WithAPIv1_1` to 0.12008, which no longer depends on `Crypt::SSLeay` (the final macOS pain point) 😄